packages/buefy-next/src/components/rate/Rate.vue

```diff
@@ -89,14 +96,15 @@ export default {
         showMe() {
             let result = ''
             if (this.showScore) {
-                result = this.disabled ? this.modelValue : this.newValue
-                if (result === 0) {
+                result = this.disabled ? this.modelValue.toString() : this.newValue.toString()
+                if (Number(result) === 0) {
                     result = ''
                 } else {
-                    result = new Intl.NumberFormat(this.locale).format(this.modelValue)
+                    result = new Intl.NumberFormat(this!.locale)
+                        .format(this.modelValue).toString()
                 }
-            } else if (this.showText) {
-                result = this.texts[Math.ceil(this.newValue) - 1]
+            } else if (this.showText && this.texts) {
+                result = this.texts[Math.ceil(this.newValue) - 1].toString()
```

Comment:
`toString` should not be necessary here.

Reply:
See https://github.com/ntohq/buefy-next/pull/285#discussion_r1787748410

Reply:
The item type of the `texts` prop is `string`, so you do not need `toString`.